### PR TITLE
Keep plugin docstring from appearing in `kedro -h`

### PR DESCRIPTION
**Pull request by deepyaman**
## Description

Partially resolves kedro-org/kedro#1749

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1001"></a>

### CODE REVIEW

File: package/kedro_viz/launchers/cli.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 @click.group(name="Kedro-Viz")
 def commands():
-    """Visualise the pipeline using Kedro viz."""
+    pass
```
Comment (deepyaman) [inline package/kedro_viz/launchers/cli.py:20]:
```suggestion
    pass  # pylint: disable=missing-method-docstring
```

File: package/kedro_viz/launchers/cli.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 @click.group(name="Kedro-Viz")
 def commands():
-    """Visualise the pipeline using Kedro viz."""
+    pass  # pylint: disable=missing-method-docstring
```
Comment (deepyaman) [inline package/kedro_viz/launchers/cli.py:20]:
```suggestion
    pass  # pylint: disable=missing-function-docstring
```

**Review by MerelTheisenQB — APPROVED** [review]

**Review by tynandebold — APPROVED** [review]

**Comment by tynandebold** [comment]
@deepyaman you can merge this anytime.

**Comment by deepyaman** [comment]
> @deepyaman you can merge this anytime.

Sorry, I just saw this (awful at managing my notifications) but thanks all for reviewing/merging!

**Pull request merged**